cheatsheets/PHP_Configuration_Cheat_Sheet.md

```diff
@@ -111,3 +107,11 @@ report_memleaks         = On
 track_errors            = Off
 html_errors             = Off
 ```
+
+## Suhosin (only usable in 5.X branch and should not be used in prod now as 5.X is EOL)
```

Comment:
We're looking to implement tables of contents for the cheatsheets at some point in the future, so this kind of long title is going to look a bit ugly when that happens. Would you be able to reformat it along the lines of:
```
## Suhosin  (PHP 5)
Don't use, unsupported, etc
[...]

## Snuggleupagus (PHP 7)
Beta, caution, etc
[...]
```

The beta warning should probably be made stronger, the suhosin page says:

"WARNING: THIS SOFTWARE IS PRE-ALPHA SOFTWARE. DO NOT ATTEMPT TO RUN IN PRODUCTION"

Reply:
Suhousin is not usable in PHP 7 at all. the link is to clearly showcase that snuggleupagus is the only option for 7.X series.

```
PHP 7.3 support and snufflepagus

For a suitable Suhosin7 replacement with PHP 7.3 support, please have a look at snufflepagus
```

Future plans of Suhousin are to improve on snuggleupagus as mentioned https://github.com/sektioneins/suhosin-ng/wiki/News Do you think this would be more appropriate link in this location.

Reply:
Suhosin-NG looks even less mature (or started) than Snuffleupagus, so I'm not sure there's too much value in linking to it at this stage. Although worth keeping an eye on how it develops.